bigbluebutton-html5/imports/ui/stylesheets/variables/general.scss

```diff
@@ -25,4 +25,6 @@
   --indicator-padding-bottom: 0.7em;
 
   --user-indicators-offset: -5px;
+
+  --enableAnimation: 1;
```

Comment:
Why are you overwriting this variable on modal style? https://github.com/bigbluebutton/bigbluebutton/pull/6634/files#diff-3b7d67cedd4b25db05c46bbaefd0a23dR2